diff -u -r ../vultures-2.1.0.orig/vultures/vultures_conf.h ./vultures/vultures_conf.h
--- ../vultures-2.1.0.orig/vultures/vultures_conf.h	2006-09-18 14:18:31.000000000 +0200
+++ ./vultures/vultures_conf.h	2006-09-18 14:23:36.000000000 +0200
@@ -8,7 +8,7 @@
 /* PCMUSIC allows us to play notes when the player uses an instrument ingame
  * nethack only defines this if __BORLANDC__ is also defined, but we can use it 
  * anyway */ 
-#define PCMUSIC
+#undef PCMUSIC
 
 /* EXPORT_TILES should be left off. This is a simple optionally built function
  * which exports as much of the Vulture's graphics as possible into a format
