--- ../vulture-2.1.0.orig/vulture/GNUmakefile
+++ ./vulture/GNUmakefile
@@ -79,7 +79,7 @@
 $(TTOUTPUT): $(BUILDDIR)/tiletrans $(TILESRC)/*.png
 	@echo "running tiletrans:"
 	@mkdir -p $(BUILDDIR)
-	@$(BUILDDIR)/tiletrans ./gamedata/graphics/gametiles.bin $(TILESRC)/
+	@$(BUILDDIR)/tiletrans ./gamedata/graphics/gametiles.bin $(TILESRC)/
 	@mv vulture_gametiles.c $(BUILDDIR)/
 	@mv vulture_gametiles.h $(BUILDDIR)/
 
